tests/ui_tests/conftest.py

```diff
@@ -67,13 +67,6 @@ def app_fs(adcm_fs: ADCM, web_driver: ADCMTest, request):
     Attach ADCM API to ADCMTest object and open new tab in browser for test
     Collect logs on failure and close browser tab after test is done
     """
-    try:
-        web_driver.new_tab()
-    # Recreate session on WebDriverException
-    except WebDriverException:
-        # this exception could be raised in case
-        # when all tabs were closed in process of creating new one
-        web_driver.create_driver()
```

Comment:
This route was meant to handle situations of unexpected driver failures. How it will be handled now?

Reply:
The comment suggests that all tabs were closed, but now it is an impossible case
I have not seen unexpected driver failures. Do you have examples?

Reply:
Nope. But additional insurance is useful)